pkgs/development/interpreters/php/generic.nix

```diff
@@ -258,6 +258,7 @@ let
             # Don't record the configure flags since this causes unnecessary
             # runtime dependencies
             ''
+              set -x
```

Comment:
did you intend to commit this?

Reply:
No, I'm trying to debug the issue we have with Darwin M1, see: https://github.com/php/php-src/issues/8101